packages/manager/modules/sms/src/sms/sms/compose/translations/Messages_fr_FR.json

```diff
@@ -115,5 +115,8 @@
   "sms_sms_compose_add_phonebook_contact_empty": "Aucun contact trouvé.",
   "sms_sms_compose_add_phonebook_contact_adding": "Mise à jour des contacts…",
   "sms_sms_compose_add_phonebook_contact_added": "Liste des contacts mise à jour !",
-  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts."
-}
\ No newline at end of file
+  "sms_sms_compose_add_phonebook_contact_ko": "Oups ! Impossible de récupérer la liste des contacts.",
+  "sms_sms_compose_message_commercial_description1": "Pour des raisons légales, un SMS à caractère commercial ne sera envoyé qu’",
```

Comment:
@Steffy29 Sorry to mention that a bit later but splitting the sentence like that isn't useful especially for a translator point of view.

Is it possible to restore the single sentence with the `<strong>` element?

Thanks!

Reply:
done